app/services/analytics.rb

```diff
@@ -166,6 +166,7 @@ def session_started_at
   OTP_SMS_INBOUND_MESSAGE_VALIDATION_FAILED = 'Twilio SMS Inbound Validation Failed'
   USER_MARKED_AUTHED = 'User marked authenticated'
   USER_REGISTRATION_AGENCY_HANDOFF_PAGE_VISIT = 'User registration: agency handoff visited'
+  USER_REGISTRATION_BACKUP_CODE_CONFIRMATION_SETUP_VISIT = 'Backup code confirmation setup visited'
```

Comment:
We should remove this constant if it's now unused after changes in 7fdb1fdeb960ca30d3f084b56affdea41993b398.

```suggestion
```